### PR TITLE
Fix scatterplot issue

### PR DESCRIPTION
**Pull request by Shine226**

### CODE REVIEW

File: wiggum_app/controller.py [hunk 1]
```diff
@@ -248,20 +252,24 @@ def main():
 
         # initial for visualize.html page
         if action == 'page_load':
+            # check if page is reload or not
+            if redirect_flag:
```
Comment (brownsarahm) [inline wiggum_app/controller.py]: What is this for? Can you add comments that describe ti in a little more detail for the sake of future maintainability

File: wiggum_app/static/js/scatterplot.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ var margin = {top: 30, right: 30, bottom: 30, left: 60},
 
 var scatterplot;
 
+// regular expression for invalid characters
```
Comment (brownsarahm) [inline wiggum_app/static/js/scatterplot.js]: invalid for what? determined by where? add some notes for maintainability

File: wiggum_app/controller.py (hunk 1 above)
Comment (Shine226) [reply wiggum_app/controller.py]: Added

File: wiggum_app/static/js/scatterplot.js (hunk 2 above)
Comment (Shine226) [reply wiggum_app/static/js/scatterplot.js]: Added

**Pull request merged**